lib/matplotlib/tests/test_determinism.py

```diff
@@ -129,10 +130,10 @@ def test_determinism_source_date_epoch(fmt, string):
     string : bytes
         Timestamp string for 2000-01-01 00:00 UTC.
     """
-    buf = subprocess.check_output(
+    buf = subprocess_run_for_testing(
         [sys.executable, "-R", "-c",
          f"from matplotlib.tests.test_determinism import _save_figure; "
          f"_save_figure('', {fmt!r})"],
         env={**os.environ, "SOURCE_DATE_EPOCH": "946684800",
-             "MPLBACKEND": "Agg"})
+             "MPLBACKEND": "Agg"}, stdout=subprocess.PIPE, text=False).stdout
```

Comment:
Not the same as above?
```suggestion
             "MPLBACKEND": "Agg"}, capture_output=True, text=False).stdout
```

Reply:
I was learning how to do this effectively as I went.